### PR TITLE
Add Asks

### PR DESCRIPTION
**Pull request by beardedtim**
This PR will add the `Asks` component.

### CODE REVIEW

File: src/features/Asks/Asks.component.test.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+import { Ask, Asks, asksWrap } from './Asks.component';
+import dom from 'cheerio';
+import serverRender from 'react-dom/server';
+import withProvider from '../../shared/testHelpers';
+
+const render = serverRender.renderToStaticMarkup;
+
+test('Ask component', assert => {
+  [
+    () => {
```
Comment (beardedtim) [inline src/features/Asks/Asks.component.test.js]: This feels hackish but did not want to spend a night or two researching `tape` in order to know how to use this in the way you are expecting.

File: src/features/Asks/Asks.component.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+export const Ask = ({ timestamp, ask, askee, status }) => (
+  <div className="ask" key={timestamp}>
+    <p className="ask_text">{ask}</p>
+    <p className="askee">{askee}</p>
+    <p className="status">{status}</p>
+  </div>
+);
+
+export const asksWrap = (children) => (
```
Comment (beardedtim) [inline src/features/Asks/Asks.component.js]: Because React still wont allow us to render arrays.

File: src/features/Home/Home.component.js [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
+import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { signIn } from './Home.reducer';
 
-const Home = () => (
-  <button className="sign-in">Sign in</button>
+
+const Home = ({ signIn }) => (
+  <div>
+    <button className="sign-in" onClick={signIn}>Sign in</button>
```
Comment (beardedtim) [inline src/features/Home/Home.component.js]: This does not update our state, even after going through reducers.

File: src/features/Asks/Asks.component.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import R from 'ramda';
+
+if (process.env.BROWSER) {
+  require('./Asks.style.css');
+}
+
+export const Ask = ({ timestamp, ask, askee, status }) => (
+  <div className="ask" key={timestamp}>
```
Comment (ericelliott) [inline src/features/Asks/Asks.component.js]: I tend to key on ids (I use `cuid()` to generate collision-safe IDs on clients and in Node).

File: src/features/Asks/Asks.component.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+export const asksWrap = (children) => (
+  <div className="asks">
+    {children}
+  </div>
+);
+
+export const Asks = R.compose(
+  asksWrap,
+  R.map(Ask),
+  R.propOr([], 'asks')
```
Comment (ericelliott) [inline src/features/Asks/Asks.component.js]: Cool!

**Pull request merged**